FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -166,23 +166,39 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </objectDesc>
                          
                         <bindingDesc>
-                            <binding contemporary="partly"></binding>
+                            <binding contemporary="partly">
                             <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote>    
                             <decoNote xml:id="b2" type="SewingStations">4</decoNote>
                             <decoNote xml:id="b3" type="bindingMaterial"><material key="wood"></material></decoNote>
+                            </binding>
                         </bindingDesc>
                        
                     </physDesc>
-                    <history><provenance>Scribe <persName>አርሳንዮስ፡</persName> mentioned in <locus target="coloph1"/> and <locus target="coloph2"></locus></provenance></history>
-                    <msPart xml:id="p1"><msIdentifier><msContents>
-                        <layoutDesc></layoutDesc>
-                        <summary></summary><msContents/><msItem xml:id="p1_i1">
+                    <history><provenance>Scribe <persName ref="PRS0000">አርሳንዮስ፡</persName> mentioned in <locus target="coloph1"/> and <locus target="coloph2"></locus></provenance></history>
+                    <additional>        
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Marrassini1986Laurenziana"/>
+                                            <citedRange unit="page">93-97</citedRange>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                    <msPart xml:id="p1"><msIdentifier/>
```

Comment:
You had `<msIdentifier>`, that is you opened the tag, but never closed it. To close an empty tag, add slash at the end `<msIdentifier/>`
You might want to rewatch the introduction to XML video